src/app/pages/Collectibles/CollectiblePage.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { FC, useState } from "react";
+
+import classNames from "clsx";
+
+import PageLayout from "app/layouts/PageLayout";
+import { navigate } from "lib/woozie";
+
+import { T } from "../../../lib/i18n/react";
+import { formatImgUri } from "../../../lib/image-uri";
+import {
+  useAccount,
+  useAssetMetadata,
+  useBalance,
+} from "../../../lib/temple/front";
+import useCopyToClipboard from "../../../lib/ui/useCopyToClipboard";
+import CopyButton from "../../atoms/CopyButton";
+import Divider from "../../atoms/Divider";
+import FormSubmitButton from "../../atoms/FormSubmitButton";
+import HashShortView from "../../atoms/HashShortView";
+import { ReactComponent as CollectiblePlaceholderLarge } from "../../icons/collectiblePlaceholderLarge.svg";
+import { ReactComponent as CopyIcon } from "../../icons/copy.svg";
+
+interface Props {
+  assetSlug: string;
+}
+
+const CollectiblePage: FC<Props> = ({ assetSlug }) => {
+  const [isLoaded, setIsLoaded] = useState(false);
+
+  const account = useAccount();
+  const accountPkh = account.publicKeyHash;
+  const { data: collectibleBalance } = useBalance(assetSlug, accountPkh);
+
+  const { copy } = useCopyToClipboard();
+  const collectibleData = useAssetMetadata(assetSlug)!;
+
+  const [, rawAssetId] = assetSlug.split("_");
```

Comment:
use `fromAssetSlug`

Reply:
`fromAssetSlug()` function is async.
I have already added a separate function `fromAssetSlugForce()` to this brunch that is synchronous.